rotkehlchen/globaldb/updates.py

```diff
@@ -267,7 +271,7 @@ def _parse_evm_token_data(
             insert_text=insert_text,
         )
         if chain_value is not None:
-            chain_id = ChainID(chain_value)
+            chain_id = ChainID.deserialize(chain_value)
```

Comment:
the potential `DeserializationError` here is already handled from what I see, right?